k4Reco/GaudiLumiCalClusterer/include/GlobalMethodsClass.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2020-2024 Key4hep-Project.
+ *
+ * This file is part of Key4hep.
+ * See https://key4hep.github.io/key4hep-doc/ for further info.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef k4RECO_GLOBALMETHODSCLASS_H
+#define k4RECO_GLOBALMETHODSCLASS_H 1
+
+#include <edm4hep/Cluster.h>
+#include <edm4hep/ReconstructedParticle.h>
+#include <edm4hep/Vector3f.h>
+
+#include <map>
+#include <string>
+#include <tuple>
+#include <variant>
+
+class LCCluster;
+class TGeoHMatrix;
+
+class GlobalMethodsClass {
```

Comment:
This class has a mixture of
- `_`-prefixed and non-prefixed member variables
- member functions starting with uppercase and lowercase letters

I think we should consistently use `m_` prefixed member variables and member functions starting with lowercase letters

Reply:
Now `m_` is used, removed `method` that apparently was unused and member functions in this class are starting with lower case. If I see more I'll change them...